examples/online/index.js

```diff
@@ -1,7 +1,11 @@
-// first:
-// $ npm install redis online
+// first (if you have not done it yet):
+// $ npm install redis
+// $ brew install redis
```

Comment:
If install instructions are added, can they be more accessible to the general user base? For example, this only works if the user is both (a) using MacOS and (b) has Homebrew installed (not of which is noted in here, just like how installing Redis was not noted before). Perhaps remove the command altogether and just add a link to Redis's site for how to install?